src/soep_preparation/clean_existing_variables/hwealth.py

```diff
@@ -0,0 +1,174 @@
+    )
+    wide["tmp_hh_gross_overall_wealth_c"] = apply_smallest_float_dtype(
+        raw_data["w010hc"]
+    )
+    wide["tmp_hh_gross_overall_wealth_d"] = apply_smallest_float_dtype(
+        raw_data["w010hd"]
+    )
+    wide["tmp_hh_gross_overall_wealth_e"] = apply_smallest_float_dtype(
+        raw_data["w010he"]
+    )
+
+    wide["tmp_hh_net_overall_wealth_a"] = apply_smallest_float_dtype(raw_data["w011ha"])
+    wide["tmp_hh_net_overall_wealth_b"] = apply_smallest_float_dtype(raw_data["w011hb"])
+    wide["tmp_hh_net_overall_wealth_c"] = apply_smallest_float_dtype(raw_data["w011hc"])
+    wide["tmp_hh_net_overall_wealth_d"] = apply_smallest_float_dtype(raw_data["w011hd"])
+    wide["tmp_hh_net_overall_wealth_e"] = apply_smallest_float_dtype(raw_data["w011he"])
+
+    wide["tmp_hh_vehicles_value_a"] = apply_smallest_int_dtype(raw_data["v010ha"])
+    wide["tmp_hh_vehicles_value_b"] = apply_smallest_int_dtype(raw_data["v010hb"])
+    wide["tmp_hh_vehicles_value_c"] = apply_smallest_int_dtype(raw_data["v010hc"])
+    wide["tmp_hh_vehicles_value_d"] = apply_smallest_int_dtype(raw_data["v010hd"])
+    wide["tmp_hh_vehicles_value_e"] = apply_smallest_int_dtype(raw_data["v010he"])
+
+    wide["tmp_hh_gross_overall_wealth_including_vehicles_a"] = (
+        apply_smallest_float_dtype(raw_data["n010ha"])
+    )
+    wide["tmp_hh_gross_overall_wealth_including_vehicles_b"] = (
+        apply_smallest_float_dtype(raw_data["n010hb"])
+    )
+    wide["tmp_hh_gross_overall_wealth_including_vehicles_c"] = (
+        apply_smallest_float_dtype(raw_data["n010hc"])
+    )
+    wide["tmp_hh_gross_overall_wealth_including_vehicles_d"] = (
+        apply_smallest_float_dtype(raw_data["n010hd"])
+    )
+    wide["tmp_hh_gross_overall_wealth_including_vehicles_e"] = (
+        apply_smallest_float_dtype(raw_data["n010he"])
+    )
+
+    wide["tmp_hh_net_overall_wealth_including_vehicles_and_student_loans_a"] = (
+        apply_smallest_float_dtype(raw_data["n011ha"])
+    )
+    wide["tmp_hh_net_overall_wealth_including_vehicles_and_student_loans_b"] = (
+        apply_smallest_float_dtype(raw_data["n011hb"])
+    )
+    wide["tmp_hh_net_overall_wealth_including_vehicles_and_student_loans_c"] = (
+        apply_smallest_float_dtype(raw_data["n011hc"])
+    )
+    wide["tmp_hh_net_overall_wealth_including_vehicles_and_student_loans_d"] = (
+        apply_smallest_float_dtype(raw_data["n011hd"])
+    )
+    wide["tmp_hh_net_overall_wealth_including_vehicles_and_student_loans_e"] = (
+        apply_smallest_float_dtype(raw_data["n011he"])
+    )
+
+    wide["tmp_flag_netwealth"] = object_to_str_categorical(
+        raw_data["n022h0"],
+        renaming={
+            "[0] No imputation": "No imputation",
+            "[1] Edited": "Edited",
+            "[2] Imputed": "Imputed",
+        },
+        ordered=True,
+    )
+
+    # Transforming the wide data to long format
+    # (the following columns were in wide format)
+    prev_wide_cols = [
```

Comment:
Here I am on the fence since they are so many long ones, but let's still include directly in the wide_to_long call (won't mention again, just look through all those=